Eerste commit, project geinitialiseerd.

LiamAHKisters opened the pull request:
Ik heb een bcrypt error die opgelost moet worden.

Commit:
Eerste commit, project geinitialiseerd.

Ik heb een bcrypt error die opgelost moet worden.

Commit:
Probleem opgeslot, controllers gegenereerd.

Bcrypt probleem is opgelost, events en reservations controllers zijn aangemaakt. De database is naar Sqlite3 verandert wegens postgres niet op Windows werkt.

Commit:
Geprobeerd te Google-api synchronizeren.

Ik heb door mijn vorige werk aan Fabmanager-zld te proberen het toe te passen voor in de Events controller, het werkt nog niet!

Commit:
Update van de index van Events_controller.

Lubmes commented on app/controllers/api/v1/events_controller.rb:
```diff
@@ -11,7 +11,8 @@ class EventsController < ApplicationController
 
 
       def index
-
```
Moeten de require's bovenaan niet in de EventsController staan?

LiamAHKisters replied:
Yup ze staan er zo ook in

Lubmes replied:
Ja dat zag ik. Ik bedoel dus andersom ;) in de class EventsController (...) end

ghost commented:
Vandaar!


________________________________
Van: Lubmes <notifications@github.com>
Verzonden: vrijdag 13 oktober 2017 13:18
Aan: Lubmes/fabmanager-zld-api
CC: LiamAHKisters; Author
Onderwerp: Re: [Lubmes/fabmanager-zld-api] Eerste commit, project geinitialiseerd. (#1)


@Lubmes commented on this pull request.

________________________________

In app/controllers/api/v1/events_controller.rb<https://github.com/Lubmes/fabmanager-zld-api/pull/1#discussion_r144527396>:

> @@ -11,7 +11,8 @@ class EventsController < ApplicationController


       def index
-


Ja dat zag ik. Ik bedoel dus andersom ;) in de class EventsController (...) end

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/Lubmes/fabmanager-zld-api/pull/1#discussion_r144527396>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AOT1YBNHVtJPpVuOSyqhSc0SvuqDC2Lrks5sr0bxgaJpZM4PqZkT>.

Commit:
Opzet reservationscontroller.

De error is opgelost de benaming van API moest niet in all caps worden gedaan. Daarnaast zijn de routes verandert en is er op Stackoverflow een vraag gesteld sinds authenticatie met de google api niet vloeiend loop.

Eventscontroller is pending.

Reservationscontroller kan aan verder gewerkt worden.

Commit:
Reservations mailer toegevoegd.

De Reservations create mail werkt, nu het afmaken van de deny mail en aan het usage van machines tijdens de open inloop.

Commit:
Afgewezen mail toegevoegd.

Mailtje voor afgewezen reserveringen is toegevoegd en een begin is gemaakt aan de Open Inloop, Control panel stuff.

Commit:
denied_mail toegevoegt aan preview

Commit:
approve_mail toegevoegt

Commit:
Reservationsmailer voltooid.

Verder nu met de Controlpanel controller.

Commit:
Gewerkt aan de control panel, moet nog stuk meer aan gedaan worden.

Commit:
Auto stash before merge of "Liam-Internship" and "origin/Liam-Internship"
